hazelcast/src/main/java/com/hazelcast/spi/impl/executionservice/ExecutionService.java

```diff
@@ -86,6 +86,12 @@ public interface ExecutionService {
      */
     String MAP_STORE_OFFLOADABLE_EXECUTOR = "hz:map-store-offloadable";
 
+    /**
+     * Name of the offloadable executor for offloaded
+     * tiered store compaction executions.
+     */
+    String MAP_TIERED_STORE_COMPACTION_OFFLOADABLE_EXECUTOR = "hz:map-tiered-store-compaction-offloadable";
```

Comment:
Will this executor run only the compactor related tasks or the regular tstore ops will be executing in this executor as well? In the latter case, consider removing "-compaction".

How this executor will be sized? Will it have a fixed amount of threads - matching the operation threads perhaps - or will create threads on demand?

Reply:
it is only compactor related ones it will execute.

It is currently used cached thread pool with its default setting, it has 16 workers by default, no auto re-sizing exists.

Reply:
So the regular ops will still run on the partition threads, right? If I understand it right, then we'll have a parallel, but not concurrent compaction wrt to map operations, and wrt compaction as well. By the latter I mean there will be no multiple threads running compaction belonging to the same hybrid log.